set.mm

```diff
@@ -385304,6 +385488,51 @@ the complex square root function (the branch cut is in the same place for
       DYOQYIYJVYIBVYRUWJYIUXGUWQVYJBAUXFDGYSYPUWKYRUWL $.
   $}
 
+  $( ~ zsqrtelqelz generalized to positive integer roots.  (Contributed by
+     Steven Nguyen, 6-Apr-2023.) $)
+  zrtelqelz $p |- ( ( A e. ZZ /\ N e. NN /\ ( A ^c ( 1 / N ) ) e. QQ ) ->
+                    ( A ^c ( 1 / N ) ) e. ZZ ) $=
+    ( cz wcel cn c1 cdiv co ccxp cdenom cfv wceq 3ad2ant3 cexp 3ad2ant1 syl2anc
+    cq syl wb qden1elz w3a qdencl nnrpd crp 1rp a1i simp2 nnzd 1exp zcn cxproot
+    cc fveq2d zq ibir eqtrd cn0 simp3 nnnn0d denexp 3eqtr2rd exp11nnd mpbid ) A
+    CDZBEDZAFBGHIHZQDZUAZVFJKZFLZVFCDZVHVIFBVHVIVGVDVIEDVEVFUBMUCFUDDVHUEUFVDVE
+    VGUGZVHFBNHZFVFBNHZJKZVIBNHZVHBCDVMFLVHBVLUHBUIRVHVOAJKZFVHVNAJVHAULDZVEVNA
+    LVDVEVRVGAUJOVLABUKPUMVDVEVQFLZVGVDVSVDAQDVSVDSAUNATRUOOUPVHVGBUQDVOVPLVDVE
+    VGURVHBVLUSVFBUTPVAVBVGVDVJVKSVEVFTMVC $.
+
+  $( A positive integer root divides its integer.  (Contributed by Steven
+     Nguyen, 6-Apr-2023.) $)
+  zrtdvds $p |- ( ( A e. ZZ /\ N e. NN /\ ( A ^c ( 1 / N ) ) e. NN ) ->
+                    ( A ^c ( 1 / N ) ) || A ) $=
+    ( cz wcel cn c1 cdiv ccxp w3a cexp cdvds wbr nnz 3ad2ant3 iddvdsexp syl2anc
+    co simp2 cc wceq zcn 3ad2ant1 cxproot breqtrd ) ACDZBEDZAFBGQHQZEDZIZUGUGBJ
+    QZAKUIUGCDZUFUGUJKLUHUEUKUFUGMNUEUFUHRZUGBOPUIASDZUFUJATUEUFUMUHAUAUBULABUC
+    PUD $.
+
+  $( NB: With more infrastructure, this could be generalized to any positive
+     integer ` A ` where ` N ` does not divide the greatest common divisor of
```

Comment:
` A ` instead of ` P `?

Reply:
My reasoning was, in the context of the theorem below it, `P` is a prime, not specifically any positive integer. The theorems around rtprmirr generally use `A`.

But I see what you mean, I'll change `this could be generalized` --> `` ` P ` in ~ rtprmirr could be generalized `` in my next pr